sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -41,6 +41,26 @@
       :title="studentMenu.applicationStatus.title"
       @click="studentMenu.applicationStatus.command"
     />
+    <v-list v-if="applicationHistory.length" density="compact" nav>
+      <v-list-group
+        v-for="application in applicationHistory"
+        :key="application.title"
+        :prepend-icon="application.props?.prependIcon"
+        :title="application.title"
+        :value="application.title"
+      >
+        <template #activator="{ props }">
+          <v-list-item v-bind="props" :title="application.title"></v-list-item>
+        </template>
+        <v-list-item
+          v-for="child in application.children"
```

Comment:
I am able to see that, when the scrollbar appears in nav(which occupies some width), the time (counter 1 in screenshot) is hidden in the group headers. If we remove the Icon(counter 2 in screenshot), it will solve this problem. Is the icon really essential for business?

![image](https://github.com/user-attachments/assets/8d13db9c-7791-4a6b-8553-0961f046ad15)